### DISCUSSION TITLE
I built a Todo app using fresh and Postgres

### DISCUSSION THREAD

**Discussion by yoni-rapoport** [freshframework/fresh#560] [Show and tell]
Tutorial (4 min read):
https://dev.to/yonirapoport/build-a-full-stack-crud-app-using-denos-fresh-and-postgres-4l39

Code:
https://github.com/yoni-rapoport/fresh-remult-todo

**Comment by ivan-kleshnin** [comment]
@yoni-rapoport thanks! Which editor do you use and how was your coding experience with Deno while doing that? I'm using Webstorm and Deno + Fresh support is very poor, tons of bugs 😢
